scripts/rollup/modules.js

```diff
@@ -52,6 +52,17 @@ const devOnlyFilesToStubOut = [
   "'ReactTestUtils'",
 ];
 
+// Ordering of these imports is important;
+// The default import must follow deep imports or Rollup breaks.
```

Comment:
@gaearon I added a comment below, we don't need this logic here :) this should address what @spicyj said as well